zsl/_state.py

```diff
@@ -0,0 +1,14 @@
+_current_app = None
```

Comment:
That's a good point. Although this is not for a discussion here I mention this.

The primary concern of app is not to inject, maybe containers could be decoupled from app and inject could be container based.

THX